vspd.go

```diff
@@ -74,6 +82,31 @@ func run(ctx context.Context) error {
 	}
 	defer db.Close()
 
+	//Check if adminPass Hash is set in db.
+	hash, err := db.GetAdminHash()
+
+	//Ensure adminpass option is set
+	if cfg.AdminPass == "" && err != nil {
```

Comment:
The comment above already tells what we're doing here, this comment can go. There should ideally be no line space between error assignment and check.
```suggestion
	hash, err := db.GetAdminHash()
	if cfg.AdminPass == "" && err != nil {
```